data/Dockerfiles/letsencrypt/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine
+
+RUN apk add --update \
+    bash \
+    curl \
+    openssl \
+    && rm -rf /var/cache/apk/*
+
+RUN curl -L https://github.com/lukas2511/dehydrated/archive/master.tar.gz \
+  | tar -xzf -
+
+ADD root/ /
```

Comment:
`COPY` is prefered over `ADD`.
https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy